.github/workflows/stale.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/stale@v4
         with:
-          stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 7 days.'
+          stale-pr-message: 'This PR was marked stale due to lack of activity. Remove stale label or this will be closed in 7 days. Commenting or Pushing will instruct the bot to automatically remove the label.'
```

Comment:
do indicate that commenting/pushing does not instantly remove the stale, and that it could take up to a day...

Reply:
probably best to avoid "Remove stale label", as only approvers will have that permission.

Reply:
Updated:

> This PR was marked stale due to lack of activity and will be closed in 7 days. Commenting or Pushing will instruct the bot to automatically remove the label. This bot runs once per day.